src/sentry/sentry_metrics/configuration.py

```diff
@@ -17,6 +17,7 @@
 class UseCaseKey(Enum):
     RELEASE_HEALTH = "release-health"
     PERFORMANCE = "performance"
+    EXPERIMENRAL = "experimental"
```

Comment:
small nit, I think it should be `EXPERIMENTAL`

Reply:
Good catch, thanks!